templates/terraform/schema_property.erb

```diff
@@ -32,14 +32,14 @@
 <% if force_new?(property, object) -%>
   ForceNew: true,
 <% end -%>
-<% unless property.validation.nil? -%>
+<% unless property.validation.nil? || property.output -%>
```

Comment:
From what I understand, `property.output` is only `Computed` and `property.default_from_api` is a `Computed` and `Optional` value so they should still get validation?

Reply:
I guess I'm worried that if a property is both marked as output and default_to_api (idk if its possible but I think it is and I could see someone doing this by mistake), this will cause issues.

Reply:
Since marking something as both `output` and `default_from_api` should be wrong, I don't think it's an issue - a bad config will produce bad results. If it comes up we can make sure it isn't optional with a big ugly conditional, but before then I don't really see the point in guarding against it.

Unfortunately I also can't see an easy place to drop a validation that both fields aren't set like we do for `output` and `required`. It looks like there's no protection against setting `default_from_api` and `required` together as well.